portal/views/assessment_engine.py

```diff
@@ -602,11 +602,19 @@ def get_assessments():
     tags:
       - Assessment Engine
     parameters:
+      - name: format
+        in: query
+        description: format of file to download (CSV or JSON)
```

Comment:
Should be fixed, not sure why Github isn't marking it as changed (maybe I added it too many lines below the referenced lines?)